src/App.js

```diff
@@ -2,12 +2,89 @@ import React, { useState } from 'react';
 import { createMuiTheme, CssBaseline } from '@material-ui/core';
 import { ThemeProvider } from '@material-ui/styles';
 import color from '@edma/design-tokens/js/color';
+import fontSize from '@edma/design-tokens/js/fontSize';
```

Comment:
Not a blocker, but we can update this later with the new import method: `import { color, font, fontSize, weight, spacing } from '@edma/design-tokens/';`